validation-test/Sema/rdar34333874.swift

```diff
@@ -0,0 +1,24 @@
+// RUN: rm -rf %t
+// RUN: mkdir -p %t
+// RUN: %target-build-swift %s -o %t/a.out
+// RUN: %target-run %t/a.out
```

Comment:
Drive-by comment: This should be `%target-run-simple-swift`, which gets run in more situations.

Reply:
Thanks, @jrose-apple ! I didn't know that existed...